cmd/storage-rest-client.go

```diff
@@ -215,17 +216,43 @@ func (client *storageRESTClient) NSScanner(ctx context.Context, cache dataUsageC
 	defer xhttp.DrainBody(respBody)
 	pr.CloseWithError(err)
 	if err != nil {
+		pr.CloseWithError(err)
 		return cache, err
 	}
+	pr.Close()
```

Comment:
This is redundant since right before if err != nil { we already have pr.CloseWithError(err)